scripts/ci/get_image_tag_from_branch.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
```

Comment:
`get_image_tag_from_branch.sh` -> `get_image_tags_from_branch.sh`

Reply:
`driver_images_tag` -> `driver_image_tags`

Reply:
> `driver_images_tag` -> `driver_image_tags`

but this one image tag for multiple driver images

Reply:
> `get_image_tag_from_branch.sh` -> `get_image_tags_from_branch.sh`

done

Reply:
regarding [this](https://github.com/IBM/ibm-block-csi-driver/pull/344#discussion_r676404852), I'm referring to [this one](https://github.com/IBM/ibm-block-csi-driver/blob/7e6c3947197125ced28694cc40b96e72d39f8140/scripts/ci/github_actions/driver/setup_ci_dependencies.sh#L20) - there is more than 1 tag here (for each image)

Reply:
done